voxblox/include/voxblox/simulation/simulation_world.h

```diff
@@ -49,6 +49,12 @@ class SimulationWorld {
   FloatingPoint getDistanceToPoint(const Point& coords,
                                    FloatingPoint max_dist) const;
 
+  // Set and get the map generation and display bounds.
```

Comment:
The current defaults for these bounds seem a bit arbitrary, maybe they should be arguments to the initialization?